docs/source/concepts/asset-model.rst

```diff
@@ -220,13 +220,14 @@ We start by modeling a standard cash bank account. There are three parties invol
 a Commercial Bank, and a Retail Client.
 
 The Central Bank defines the economic terms of the currency asset and is generally a highly trusted
-entity; hence, it acts as ``issuer`` as well as ``depository`` of the corresponding instrument.
+entity. Hence, it acts as ``issuer`` as well as ``depository`` of the corresponding instrument.
```

Comment:
The `;` is *more* correct in this case as the first sentence requires the second sentence (ie., the second sentence cannot standalone without the first sentence and a `;` is what you use to show that). 

Or we can make a minor change if you don't like the semicolon :

```suggestion
entity, therefore it acts as ``issuer`` as well as ``depository`` of the corresponding instrument.
```